lte/gateway/c/oai/tasks/mme_app/mme_app_bearer.c

```diff
@@ -583,7 +583,13 @@ void mme_app_handle_initial_ue_message(mme_app_desc_t *mme_app_desc_p,
 
   OAILOG_INFO(LOG_MME_APP, "Received MME_APP_INITIAL_UE_MESSAGE from S1AP\n");
 
-  DevAssert(INVALID_MME_UE_S1AP_ID == initial_pP->mme_ue_s1ap_id);
+  if (INVALID_MME_UE_S1AP_ID != initial_pP->mme_ue_s1ap_id) {
```

Comment:
Move the constant to the right side of comparison, i.e. 
`if (initial_pP->mme_ue_s1ap_id != INVALID_MME_UE_S1AP_ID)`

Reply:
I have committed the change as per the review comment